### ISSUE TITLE
Interpolation bug

### ISSUE THREAD

**Issue by rohan-shah** [jackparmer/colorlover#17]
```
cl.interp(cl.scales['9']['seq']['Reds'], 100) #Works
cl.interp(cl.scales['9']['seq']['Reds'], 101) #Exception
cl.interp(cl.scales['9']['seq']['Reds'], 110) #Works
```

I don't see why this should happen.

**Comment by rambo-yuanbo** [comment]
i ran into this bug, too.

**Comment by ancantus** [comment]
Reproduced for me as well.
```
.../venv/local/lib/python2.7/site-packages/colorlover/__init__.pyc in interp(scl, r)
   1821         c_i = int(i*math.floor(SCL_FI)/round(r[-1])) # start color index
   1822         hsl_o = rgb_to_hsl( scl[c_i] ) # convert rgb to hls
-> 1823         hsl_f = rgb_to_hsl( scl[c_i+1] )
   1824         section_min = c_i*r[-1]/SCL_FI
   1825         section_max = (c_i+1)*(r[-1]/SCL_FI)

IndexError: list index out of range
```

**Comment by jonathanrocher** [comment]
Anyone found a fix for this issue?

**Comment by jonmmease** [comment]
The `IndexError` was fixed by #9 and will be released in version 0.3.

The issue with colors not being interpolated smoothly will be tracked in https://github.com/jackparmer/colorlover/issues/18
